sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM `spell_script_names` WHERE `spell_id`=12294 AND `ScriptName`='spell_warr_mortalstrike';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (12294, 'spell_warr_mortalstrike');
+
+DELETE FROM `spell_script_names` WHERE `spell_id`=167105 AND `ScriptName`='spell_warr_colossus_smash';
```

Comment:
Use this format for your `DELETE FROM` commands:
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_warr_mortal_strike';
DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_warr_colossus_smash';
```
The logic here is to make sure you get rid of other spell scripts with the same name (if they exist), but with different (wrong) spell IDs.

As a matter of fact, you could simplify the SQL like this:
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_warr_mortal_strike','spell_warr_colossus_smash');
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
( 12294, 'spell_warr_mortal_strike'),
(167105, 'spell_warr_colossus_smash');
```
You can of course keep your 2 `DELETE FROM` commands if you like, if you find that easier to view or if you want to add more later. Just keep in mind that you only need 1 `INSERT INTO` command for the same table. :-)